trickle.go

```diff
@@ -0,0 +1,66 @@
+package kapacitor
+
+import (
+	"github.com/influxdata/kapacitor/edge"
+	"github.com/influxdata/kapacitor/models"
+	"github.com/influxdata/kapacitor/pipeline"
+)
+
+func newTrickleNode(et *ExecutingTask, n *pipeline.TrickleNode, d NodeDiagnostic) *TrickleNode {
+	sn := &TrickleNode{
+		node: node{Node: n, et: et, diag: d},
+	}
+	sn.node.runF = sn.runTrickle
+	return sn
+}
+
+type TrickleNode struct {
+	node
+	dims models.Dimensions
+	name string
+}
+
+func (n *TrickleNode) runTrickle(_ []byte) error {
+	consumer := edge.NewConsumerWithReceiver(
+		n.ins[0],
```

Comment:
Do we enforce anywhere that `len(ins) == 1`?

Reply:
Yes, in the attachment, in the tick code.